manifests/params.pp

```diff
@@ -16,54 +29,73 @@
   $composer_max_age    = 30
   $pear_ensure         = 'present'
   $pear_package_suffix = 'pear'
-  $phpunit_source    = 'https://phar.phpunit.de/phpunit.phar'
-  $phpunit_path      = '/usr/local/bin/phpunit'
-  $phpunit_max_age   = 30
+  $phpunit_source      = 'https://phar.phpunit.de/phpunit.phar'
+  $phpunit_path        = '/usr/local/bin/phpunit'
+  $phpunit_max_age     = 30
 
   case $::osfamily {
+
     'Debian': {
-      $config_root             = pick($cfg_root, '/etc/php5')
+
+      if $::lsbdistcodename == 'wheezy' {
+        $manage_repos = true
+      } elsif $::operatingsystem == 'Ubuntu' {
+        $manage_repos = true
+      } else {
+        $manage_repos = false
+      }
+
+      if $::lsbdistcodename == 'stretch' or $::lsbdistcodename == 'xenial' {
```

Comment:
Ah I see. Even though the lsbxxx facts are commonly used everywhere, they depend on the lsb-release package. We could change it to use the os hash but would need to retest.